arcgis-ios-sdk-samples/Features/Display subtype feature layer/DisplaySubtypeFeatureLayer.storyboard

```diff
@@ -0,0 +1,244 @@
+        <scene sceneID="1ba-C4-kqP">
+            <objects>
+                <viewController id="ivQ-zm-dUQ" userLabel="Display Subtype Feature Layer View Controller" customClass="DisplaySubtypeFeatureLayerViewController" customModule="ArcGIS_Runtime_SDK_Samples" customModuleProvider="target" sceneMemberID="viewController">
+                    <view key="view" contentMode="scaleToFill" id="SeK-4v-nM4">
+                        <rect key="frame" x="0.0" y="0.0" width="414" height="896"/>
+                        <autoresizingMask key="autoresizingMask" widthSizable="YES" heightSizable="YES"/>
+                        <subviews>
+                            <view contentMode="scaleToFill" translatesAutoresizingMaskIntoConstraints="NO" id="sLt-8I-td0" userLabel="Map View" customClass="AGSMapView">
+                                <rect key="frame" x="0.0" y="0.0" width="414" height="896"/>
+                                <color key="backgroundColor" systemColor="systemBackgroundColor" cocoaTouchSystemColor="whiteColor"/>
+                            </view>
+                            <toolbar opaque="NO" clearsContextBeforeDrawing="NO" contentMode="scaleToFill" translatesAutoresizingMaskIntoConstraints="NO" id="Jcp-Sk-bVa">
+                                <rect key="frame" x="0.0" y="813" width="414" height="49"/>
+                                <items>
+                                    <barButtonItem style="plain" systemItem="flexibleSpace" id="OtO-Lk-i54"/>
+                                    <barButtonItem title="Settings" id="Qe7-1R-wDr">
+                                        <connections>
+                                            <segue destination="uXD-44-Rhn" kind="popoverPresentation" popoverAnchorBarButtonItem="Qe7-1R-wDr" id="Iom-2Q-kpX">
+                                                <popoverArrowDirection key="popoverArrowDirection" up="YES" down="YES" left="YES" right="YES"/>
+                                            </segue>
+                                        </connections>
+                                    </barButtonItem>
+                                    <barButtonItem style="plain" systemItem="flexibleSpace" id="Oxd-KQ-o9a"/>
+                                </items>
+                            </toolbar>
+                        </subviews>
+                        <color key="backgroundColor" systemColor="systemBackgroundColor" cocoaTouchSystemColor="whiteColor"/>
+                        <constraints>
+                            <constraint firstItem="Jcp-Sk-bVa" firstAttribute="bottom" secondItem="WUM-OG-Ph5" secondAttribute="bottom" id="GFw-At-N9o"/>
+                            <constraint firstItem="Jcp-Sk-bVa" firstAttribute="trailing" secondItem="WUM-OG-Ph5" secondAttribute="trailing" id="JJJ-d3-yxB"/>
+                            <constraint firstAttribute="bottom" secondItem="sLt-8I-td0" secondAttribute="bottom" id="XPD-lB-MWL"/>
```

Comment:
Please constrain the bottom of the map view to the top of the toolbar so that attribution bar is visible.